rllib/evaluation/tests/test_rollout_worker.py

```diff
@@ -708,6 +708,38 @@ def test_wrap_multi_agent_env(self):
         self.assertTrue(isinstance(ev.env, VideoMonitor))
         ev.stop()
 
+    def test_no_training(self):
```

Comment:
this isn't an end to end test. I've tested running a client and a server and checking that training doesn't kick in, but that would be great to have a unit test for this.
Something like:

```python
# no training
start_episode(episode_id, training_enabled=False)
obs = obs_space.sample()
actions = set()
for _ in range(train_batch_size * 2):
    resp = test_server.get_action(episode_id, obs)
    actions.add(resp.json()["action"])
self.assertEqual(1, len(actions))

# training
start_episode(episode_id, training_enabled=True)
obs = obs_space.sample()
actions = set()
for _ in range(train_batch_size * 2):
    resp = test_server.get_action(episode_id, obs)
    actions.add(resp.json()["action"])
self.assertNotEqual(1, len(actions))
```